src/main/java/uk/gov/hmcts/divorce/common/event/page/Applicant2SolConfirmContactDetails.java

```diff
@@ -29,11 +29,11 @@ public void addTo(PageBuilder pageBuilder) {
             .complex(CaseData::getApplicant2)
                 .complex(Applicant::getSolicitor)
                     .mandatoryNoSummary(Solicitor::getName, null, "${labelContentApplicant2UC}'s solicitor's name")
-                    .optionalNoSummary(Solicitor::getPhone, null, "${labelContentApplicant2UC}'s solicitor's Phone number")
-                    .label("LabelRespSol-EmailHeader", "### Email contact details")
-                    .label("LabelRespSol-Email", "Email address will be used to send case updates.")
-                    .mandatoryNoSummary(Solicitor::getEmail, null, "${labelContentApplicant2UC}'s solicitor's Email")
-                    .mandatoryNoSummary(Solicitor::getAgreeToReceiveEmailsCheckbox)
+                    .optionalNoSummary(Solicitor::getPhone, null, "${labelContentApplicant2UC}'s solicitor's phone number")
+                    .label("LabelRespSol-EmailHeader", "### Email updates and service")
+                    .label("LabelRespSol-Email", "Updates on the case will be sent to this email address")
+                    .mandatoryNoSummary(Solicitor::getEmail, null, "${labelContentApplicant2UC}'s solicitor's email")
+                    .mandatoryNoSummary(Solicitor::getAgreeToReceiveEmailsCheckbox, null, " ")
```

Comment:
What is the desired outcome here? No label?

Reply:
Yeah, this is to remove the default label "Service by email" which is not required as per this doc
https://docs.google.com/document/d/1WxuEw2mRZREZwdfSpj4MsmTuahxQtjOFjwdYUfU7zDY/edit#bookmark=id.p1z0i4z3vq1

Reply:
added a constant for blank label